src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -23,11 +25,15 @@ import {
   fetchItems,
   ShoppingListItem,
   updateItem,
+  reorderItems,
 } from "../../../data/shopping-list";
 import { SubscribeMixin } from "../../../mixins/subscribe-mixin";
 import { HomeAssistant } from "../../../types";
 import { LovelaceCard, LovelaceCardEditor } from "../types";
 import { SensorCardConfig, ShoppingListCardConfig } from "./types";
+import { mdiDrag, mdiSort, mdiPlus, mdiNotificationClearAll } from "@mdi/js";
```

Comment:
Move to the top of the file under Guard import. Absolute imports should be above relative

Reply:
Good to know 👍🏽. Fixed.